tools/scdblfinder.cwl

```diff
@@ -14,7 +14,7 @@ requirements:
         entry:
           $include: ../scripts/scDblFinder.Rmd
   - class: ResourceRequirement
-    coresMin: 8
+    coresMin: $(inputs.cpus)
     ramMin: 16000
```

Comment:
Can this be made a variable as well?

Reply:
Gah! That was an oversight. Will update before merge